docs/src/main/asciidoc/deploying-to-openshift.adoc

```diff
@@ -90,19 +113,44 @@ or
 quarkus.container-image.builder=jib
 ----
 
-=== Deploying
+=== Build and Deploy (in a single step)
 
-To trigger a deployment:
+If OpenShift extension was not included during the bootstraping of the project nor added subsequently  (check pom.xml file for it), then add OpenShift as a dependency before building:
```

Comment:
```suggestion
If the OpenShift extension was not included during the bootstraping of the project nor was it added subsequently  (check pom.xml file for it), then it can be added like so:
```

Reply:
Thanks. Done.